src/@next/Pagination/PaginationStyle.tsx

```diff
@@ -58,6 +63,33 @@ export const StyledSimplePaginationButton = styled(StyledPageButton)`
   pointer-events: none;
 `;
 
+export const StyledBorderPaginationButton = styled(StyledPageButton)`
+  border: 1px solid ${Neutral.B85};
```

Comment:
@davidzzz please change the property from `border` to `outline` with value remaining the same and set the column-gap to 1px. And remove the 'border-right-style' and 'border-left-style' for the first-of-type and last-of-type